assets/components/atoms/social-icon/social-icon-discrete-link.twig

```diff
@@ -0,0 +1,9 @@
+<a href="#" class="social-icon social-icon-facebook social-icon-discrete">
+  {% include '../../atoms/icon/icon.twig' with { icon: 'icon-facebook' } %}
```

Comment:
bad include path, please use the [proper one](http://frontend.github.io/toolbox/html/twig.html#import-components)

Reply:
Getting use to the new toolbox, thx 😄